app/whats-new/whats-new.njk

```diff
@@ -1,19 +1,35 @@
 {% extends "layouts/base.njk" %}
 
 {% block pageTitle %}
-  {{ heading }} – {{ super() }}
+  What's new
 {% endblock %}
 
-{% block content %}  
-  {% if contentBody and contentTitle %}
-    <header>
-      <h1 class="govuk-heading-xl govuk-!-margin-top-1 govuk-!-margin-bottom-6">
-        {{ contentTitle }}
-      </h1>
-    </header>
+{% block content %}
+  <header>
+    <h1 class="govuk-heading-xl govuk-!-margin-top-1 govuk-!-margin-bottom-6">
+      What's new
+    </h1>
+  </header>
+
+  <p class="govuk-!-margin-bottom-5">
+    The latest updates and improvements to Book a Secure Move.
+  </p>
+
+  {% for post in posts %}
+    {% if post.fields.body and post.fields.title %}
```

Comment:
Is this `if` statement necessary? I guess it feels like the body and the title will always be set, even if it's an empty string?